.github/workflows/test.yml

```diff
@@ -10,12 +10,12 @@ jobs:
     strategy:
       matrix:
         os: [macos-latest, ubuntu-latest, windows-latest]
-        node-version: [^12.13]
+        node-version: [18.x]
```

Comment:
we may want to add a matrix to test across node versions, or at least cover the minimum specified in the node engines field.

If it can't run on an older version, and there is no suitable workaround, you can them bump the node engines version along with (ideally) a major version bump is the standard practise

Reply:
yep - one we can add in soon